SimpleITK/utilities/vtk.py

```diff
@@ -64,63 +63,52 @@ def sitk2vtk(img, debugOn=False):
             1.0,
         ]
 
+    # Create VTK image and set its metadata
+    vtk_image = vtk.vtkImageData()
     vtk_image.SetDimensions(size)
     vtk_image.SetSpacing(spacing)
     vtk_image.SetOrigin(origin)
     vtk_image.SetExtent(0, size[0] - 1, 0, size[1] - 1, 0, size[2] - 1)
-
     if vtk.vtkVersion.GetVTKMajorVersion() < 9:
-        print("Warning: VTK version <9.  No direction matrix.")
+        logging.warning(
```

Comment:
The root logger should not be used. A package specific logger should be created to allow control over logging at the package level.

Reply:
Fixed.

Reply:
The module specific logger is an OK approach too.